docs/code_contribution_guidelines.md

```diff
@@ -307,9 +308,26 @@ new breaking change to a module's API is introduced:
 - Repeat the process for any other modules the require a new major as a result
   of consuming the new major(s).
 
+<a name="#HandlingBackports" />
```

Comment:
Does not need the `#` as this does not allow you to jump here from the index.

Reply:
Updated.